src/components/Records/Search/Output/RecordsCardColumn.vue

```diff
@@ -58,7 +58,7 @@
               :disabled="Chips[item.title].length === 0"
               @click="changeActiveItem(index)"
             >
-              {{ item.title }} ({{ Chips[item.title].length }})
+              {{ item.title==='taxonomies'?'species':item.title }} ({{ Chips[item.title].length }})
```

Comment:
remember, this is JS so it's ** item.title === 'taxonomies' ? 'species' : item.title** with whitespaces around operators.

Reply:
I did approve the PR though, so you can merge anyway.